romancal/stpipe/tests/test_integration.py

```diff
@@ -25,4 +25,4 @@ def test_entry_point():
     all_steps = entry_points.get_steps()
     roman_steps = [s for s in all_steps if s.package_name == "romancal"]
     tuples = {(s.class_name, s.class_alias, s.is_pipeline) for s in roman_steps}
-    assert tuples == set(get_steps())
+    assert len(tuples) == len(set(entry_points.get_steps()))
```

Comment:
@eslavich I wasn't sure how you wanted to test this. `get_steps` returns `StepInfo` objects and comparing to `tuples` fails. Let me know if you had something else in mind.

Reply:
Did the original assertion fail?  `integration.get_steps()` should return [tuples](https://github.com/spacetelescope/romancal/blob/main/romancal/stpipe/integration.py#L23)?

Reply:
yes, but I'll double check

Reply:
If the entry point wasn't installed, that would cause a failure.  It doesn't automatically install when pulling changes down even when the package is installed with `-e` -- it has to be pip install'ed again.